runtime/onert/core/include/backend/train/LayerScopeTensor.h

```diff
@@ -26,6 +26,12 @@ namespace backend
 namespace train
 {
 
+enum class LayerScopeTensorLifeTime : unsigned char
```

Comment:
(out of my curiosity)
Why do you set a type for this enumaration class? What is the advantages?

Reply:
@jyoungyun 

AFAIK, if we didn't set a type(`enum-base`) for enum class, the enum element takes 4 bytes by default. 
In this case, I'd like to reduce the size of enum element. (default : 4bytes -> unsigned char 1bytes)